Source/EasyNetQ/Interception/GZipInterceptor.cs

```diff
@@ -12,7 +12,7 @@ public class GZipInterceptor : IProduceConsumeInterceptor
         public ProducedMessage OnProduce(ProducedMessage message)
         {
             var properties = message.Properties;
-            var body = message.Body;
+            var body = message.Body.ToArray(); // TODO Do not copy here
```

Comment:
compressingStream.Write(message.Body.Span);

Reply:
Lol, it doesn't exist in netstantard2

Reply:
You can multitarget EasyNetQ for netstandard2.0 and netstandard2.1 and use `#if` in order to unleash the full potential of the APIs for working with spans, if used on modern platforms (netcoreapp3.1 and newer).

Reply:
Sure thing, but let's do it in a separate PR.